_infra/helm/uaa/Chart.yaml

```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 1.0.0
+version: 2.0.0
```

Comment:
Should we have these match the app version like in all the other repos?  or does this one need to be different?